test/JDBC/sql_expected/BABEL-IDENTITY.out

```diff
@@ -709,6 +709,142 @@ int#!#int
+go
+~~START~~
+text
+Query Text: SELECT id, mycol FROM dbo.test_identity_index WHERE id = @@identity
+Index Scan using test_identity_index_pkey on test_identity_index  (cost=0.16..8.18 rows=1 width=8)
+  Index Cond: (id = babelfish_get_last_identity())
+~~END~~
+
+
+SELECT id, mycol FROM dbo.test_identity_index WHERE dbo.test_identity_index.id = scope_identity();
+go
+~~START~~
+text
+Query Text: SELECT id, mycol FROM dbo.test_identity_index WHERE dbo.test_identity_index.id = scope_identity()
+Index Scan using test_identity_index_pkey on test_identity_index  (cost=0.16..8.18 rows=1 width=8)
+  Index Cond: (id = babelfish_get_last_identity())
+~~END~~
+
+
+SELECT id, mycol FROM dbo.test_identity_index WHERE id > scope_identity();
+go
+~~START~~
+text
+Query Text: SELECT id, mycol FROM dbo.test_identity_index WHERE id > scope_identity()
+Bitmap Heap Scan on test_identity_index  (cost=9.99..31.29 rows=753 width=8)
+  Recheck Cond: (id > babelfish_get_last_identity())
+  ->  Bitmap Index Scan on test_identity_index_pkey  (cost=0.00..9.80 rows=753 width=0)
+        Index Cond: (id > babelfish_get_last_identity())
+~~END~~
+
+
+SELECT id, mycol FROM dbo.test_identity_index WHERE id != scope_identity();
+go
+~~START~~
+text
+Query Text: SELECT id, mycol FROM dbo.test_identity_index WHERE id != scope_identity()
+Seq Scan on test_identity_index  (cost=0.00..43.90 rows=2259 width=8)
+  Filter: (id <> babelfish_get_last_identity())
+~~END~~
+
+
+SELECT id, mycol FROM dbo.test_identity_index WHERE @@identity < id;
+go
+~~START~~
+text
+Query Text: SELECT id, mycol FROM dbo.test_identity_index WHERE @@identity < id
+Bitmap Heap Scan on test_identity_index  (cost=9.99..31.29 rows=753 width=8)
+  Filter: (babelfish_get_last_identity() < id)
+  ->  Bitmap Index Scan on test_identity_index_pkey  (cost=0.00..9.80 rows=753 width=0)
+        Index Cond: (id > babelfish_get_last_identity())
+~~END~~
+
+
+SELECT id, mycol FROM dbo.test_identity_index WHERE mycol = 10 AND id = scope_identity();
+go
+~~START~~
+text
+Query Text: SELECT id, mycol FROM dbo.test_identity_index WHERE mycol = 10 AND id = scope_identity()
+Index Scan using test_identity_index_pkey on test_identity_index  (cost=0.16..8.18 rows=1 width=8)
+  Index Cond: (id = babelfish_get_last_identity())
+  Filter: (mycol = 10)
+~~END~~
+
+
+SELECT id, mycol FROM dbo.test_identity_index WHERE id <= scope_identity() OR mycol = 11;
+go
+~~START~~
+text
+Query Text: SELECT id, mycol FROM dbo.test_identity_index WHERE id <= scope_identity() OR mycol = 11
+Seq Scan on test_identity_index  (cost=0.00..49.55 rows=761 width=8)
+  Filter: ((id <= babelfish_get_last_identity()) OR (mycol = 11))
+~~END~~
+
+
+SELECT num_index, mycol FROM dbo.test_numeric_index WHERE num_index = scope_identity();
+go
+~~START~~
+text
+Query Text: SELECT num_index, mycol FROM dbo.test_numeric_index WHERE num_index = scope_identity()
+Index Scan using test_numeric_index_pkey on test_numeric_index  (cost=0.16..8.18 rows=1 width=24)
```

Comment:
We usually don't include cost in the expected result because it is subject to change in future. 
plz check with jungkook whether there is a way to hide cost in BBF explain plan .